recipe/meta.yaml

```diff
@@ -14,12 +15,12 @@ source:
     # the quick correctness tests, without bothering with gflags (which didn't
     # work out of the box). This is the recommended method in the README.
     - skip-microbenchmarks.patch
-    # Patch CMakeLists to build a static library as well.
-    # Submitted upstream as: https://github.com/google/snappy/pull/48
-    - build-static-lib.patch
+    # Name the static library snappy_static.lib on windows, to distinguish from
+    # the dynamic library, since they have the same suffix there.
+    - windows-static-lib-name.patch
```

Comment:
Can we try upstreaming this patch? Hopefully since we are just trying to get them to change the library name, it should be easy to accept.

Reply:
Sure, I can try. :)

Reply:
https://github.com/google/snappy/pull/57

Reply:
> Sure, I can try. :)

That's all we can ask. ;) Thanks for trying. :)